babel_extractors.csv

```diff
@@ -0,0 +1,9 @@
+hooks.py,frappe.gettext.extractors.navbar.extract
```

Comment:
Is this because you don't want to rely on DB connection?

Reply:
Yes, reading `hooks.py` currently depends on a db connection and I wanted to avoid this. Reading `toml` files seemed to require an additional library. So I thought let's keep it simple and go with CSV.